src/main/kotlin/no/nav/syfo/personstatus/db/getFromPersonOversiktStatus.kt

```diff
@@ -36,24 +37,7 @@ const val queryHentUbehandledePersonerTilknyttetEnhet = """
                         SELECT *
                         FROM PERSON_OVERSIKT_STATUS
                         WHERE ((tildelt_enhet = ?)
-                            AND (motebehov_ubehandlet = 't'
-                            OR oppfolgingsplan_lps_bistand_ubehandlet = 't'
-                            OR dialogmotesvar_ubehandlet = 't'
-                            OR (
-                                dialogmotekandidat = 't'
-                                AND dialogmotekandidat_generated_at + INTERVAL '7 DAY' < now()
```

Comment:
Denne sjekken gjøres også i koden etter man har hentet fra db så tror ikke den trengs her.

Reply:
Det vil gjøre at flere rader inkluderes fra query'et, ikke sant? Og så blir de filtrert ut av Kotlin-koden.

Kan jo tenkes at det vil gjøre at det går litt treigere (siden mer data skal flyttes og objekter instansieres i Java VM'en), men det er ikke sikkert det er merkbart. Kan følge med og se om responstidene øker (feks i cloud console: som en baseline kjører nå disse query'ene på ca 3.9 ms i snitt over et døgn)

Reply:
Stemmer, men tviler på at det er merkbart for et enkelt kontor. Vi får følge med :crossed_fingers: